### ISSUE TITLE
Suggestion: ability to read config loaded from a file

### ISSUE THREAD

**Issue by benstevens48** [systemjs/builder#792]
I'd like to be able to read the systemjs config which I'm loading from a file. Looking at the code, I think I can so this using
```
builder.loader.getConfig()
```
but I'm not sure if this is officially supported. If it is supported to access the loader in this way, could it be added to the docs? Else a builder.getConfig() function would be useful.

**Comment by amenadiel** [comment]
I guess you want to read more than the original config object. So the use case is: setting the config, modifying the config, then read the resulting config.

Otherwise, you'd already have required said object from a file.

**Comment by benstevens48** [comment]
Yes it would be useful to be able to read the modified config, but even just reading the original config from a file is not so straightforward (because the config file looks like SystemJS.config({...}) and possibly contains other variable declarations etc, so really needs to be evaluated inside its own context which has a SystemJS instance called SystemJS in it.
